modules/appeals_api/app/workers/appeals_api/higher_level_review_upload_status_batch.rb

```diff
@@ -9,11 +9,6 @@ class HigherLevelReviewUploadStatusBatch
     # No need to retry since the schedule will run this every hour
     sidekiq_options retry: false, unique_for: 30.minutes
 
-    # Throttling works by grabbing a majority of the oldest records, but also some of the newest.
```

Comment:
This was created when we were transitioning HLRv1 records to the newer statuses CMP sending to bring the old records up to the newest status while still updating the newest records so that Veterans did not have to wait many hours of catch-up runs. 

No longer needed since all the records have been caught up w/ the new statuses.